src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,27 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
+    private static final String START_OF_WORDS = "w";
+    private static final String REGEX_NON_WORDS = "[^a-z+]";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> arr = new ArrayList<>();
```

Comment:
variable names should be full and informative. 
https://mate-academy.github.io/style-guides/java/java.html#s5.2.5-non-constant-field-names